codespell_lib/tests/test_dictionary.py

```diff
@@ -65,17 +68,24 @@ def test_dictionary_formatting(fname, in_aspell):
 
 
 def _check_aspell(phrase, msg, in_aspell, fname):
-    if speller is None:
+    if not spellers:  # if no spellcheckers exist
         return  # cannot check
     if in_aspell is None:
         return  # don't check
+    elif in_aspell is False:
+        language = supported_languages
+    elif in_aspell is True:
+        language = ('en',)
+    else:
+        language = in_aspell
     if ' ' in phrase:
         for word in phrase.split():
             _check_aspell(word, msg, in_aspell, fname)
         return  # stop normal checking as we've done each word above
-    this_in_aspell = speller.check(
-        phrase.encode(speller.ConfigKeys()['encoding'][1]))
-    end = 'be in aspell for dictionary %s' % (fname,)
+    this_in_aspell = any(spellers[lang].check(phrase.encode(
+        spellers[lang].ConfigKeys()['encoding'][1])) for lang in language)
+    end = 'be in aspell dictionaries %s for dictionary %s' % (
+        ' '.join(map(str, language)), fname,)
```

Comment:
Is `language` _always_ an array of strings?
If it is, I think you can replace ` ' '.join(map(str, language))` with ` ' '.join(language)` ?

(and just a personal stylistic thing, but I think arrays look better when they're named as plurals. So `languages` instead of `language` ?)

Reply:
Yes. It is.

Yes. Change it to languages came to my mind too.

I adapted both